romancal/skymatch/skymatch_step.py

```diff
@@ -52,11 +51,12 @@ class SkyMatchStep(RomanStep):
 
     def process(self, input):
         self.log.setLevel(logging.DEBUG)
-        self._is_asn = False
+        self._is_asn = False  # FIXME: where is this used?
```

Comment:
I don't think we need that for now. Maybe it was a placeholder variable that was intended to be used as an input parameter? Regardless of the initial intent, I think it is safe to remove it.

Reply:
Thanks! Removed in: c1089888393ebf83604fa0bac42b8f3e87fbae82